static/src/javascripts/projects/common/modules/commercial/user-ad-targeting.spec.js

```diff
@@ -1,21 +1,13 @@
 // @flow
 import { local } from 'lib/storage';
-import {
-    getUserFromApi as getUserFromApi_,
-    getUserFromCookie as getUserFromCookie_,
-} from 'common/modules/identity/api';
-import {
-    getUserSegments,
-    requestUserSegmentsFromId,
-} from 'common/modules/commercial/user-ad-targeting';
+import { getUserFromCookie as getUserFromCookie_ } from 'common/modules/identity/api';
+import { getUserSegments } from 'common/modules/commercial/user-ad-targeting';
 
-const getUserFromApi: any = getUserFromApi_;
 const getUserFromCookie: any = getUserFromCookie_;
 
 jest.mock('lib/storage');
 jest.mock('common/modules/identity/api', () => ({
     getUserFromCookie: jest.fn(),
-    getUserFromApi: jest.fn(),
 }));
 const userSegmentsKey = 'gu.ads.userSegmentsData';
```

Comment:
Are we able to check with @guardian/commercial-dev whether this is used any more?